nixos/modules/services/security/tor.nix

```diff
@@ -1008,7 +1008,7 @@ in
         #InaccessiblePaths = [ "-+${runDir}/root" ];
         UMask = "0066";
         BindPaths = [ stateDir ];
-        BindReadOnlyPaths = [ storeDir "/etc" ];
+        BindReadOnlyPaths = [ storeDir "/etc" ] ++ lib.optional config.services.resolved.enable "/run/systemd/resolve/stub-resolv.conf";
```

Comment:
It could also be ``/run/systemd/resolve/resolv.conf``. I think we should fix it once when not using the stub-resolve, too.

Reply:
Under which circumstances could the symlink link to `/run/systemd/resolve/resolv.conf`? I didn't find anything in Nixpkgs.

Reply:
If you are using systemd-resolved without the stub resolver because port 53 is already occupied then you can symlink resolved.conf to that file. Also if the stub-resolver is disabled ``stub-resolv.conf`` is a symlink to ``resolv.conf``.

Reply:
Should I just allow access to the entire directory? Then the `io.systemd.Resolve` socket would also be accessible. Or would it be better to just add `/run/systemd/resolve/resolv.conf` to `BindReadOnlyPaths` instead?

Reply:
I don't know. I would just add the second config file to prevent things going bad because of the socket.

Reply:
Ok, done. thanks for the suggestion.